js/oceanex.js

```diff
@@ -0,0 +1,323 @@
+        const balances = this.safeValue (response['data'], 'accounts');
+        let result = { 'info': balances };
+        for (let i = 0; i < balances.length; i++) {
+            const balance = balances[i];
+            let currency = balance['currency'];
+            let account = this.account ();
+            let free = this.safeFloat (balance, 'balance');
+            let used = this.safeFloat (balance, 'locked');
+            let total = free + used;
+            account['free'] = free;
+            account['used'] = used;
+            account['total'] = total;
+            result[currency] = account;
+        }
+        return this.parseBalance (result);
+    }
+
+    async createOrder (symbol, type, side, amount, price = undefined, params = {}) {
+        let response = await this.privatePostOrders (this.extend (params));
+        const data = response['data'];
+        let result = {
+            'info': data,
+            'id': data['id'],
+            'symbol': data['market'],
+            'type': data['ord_type'],
+            'side': data['side'],
+            'status': data['state'],
+        };
+        return result;
+    }
+
+    async fetchOrders (symbol = undefined, since = undefined, limit = undefined, params = {}) {
+        let response = await this.privateGetOrders (this.extend (params));
+        const data = response['data'];
+        return this.parseOrders (data);
+    }
+
+    parseOrder (order, market = undefined) {
+        let result = {
+            'info': order,
+            'id': order['id'],
+            'timestamp': order['created_on'],
+            'datetime': order['created_at'],
+            'lastTradeTimestamp': undefined,
+            'symbol': order['market'],
+            'type': order['ord_type'],
+            'side': order['side'],
+            'price': this.safeFloat (order, 'price'),
+            'average': this.safeFloat (order, 'avg_price'),
+            'amount': this.safeFloat (order, 'volume'),
+            'remaining': this.safeFloat (order, 'remaining_volume'),
+            'filled': this.safeFloat (order, 'executed_volume'),
+            'status': order['state'],
+            'fee': undefined,
+        };
+        return result;
+    }
+
+    async createMultiOrders (params = {}) {
+        let response = await this.privatePostOrdersMulti (this.extend (params));
+        const data = response['data'];
+        return this.parseOrders (data);
+    }
+
+    async filterOrders (params = {}) {
+        let response = await this.privateGetOrdersFilter (this.extend (params));
+        const data = response['data'];
+        return this.parseOrders (data);
+    }
+
+    async cancelOrder (id, symbol = undefined, params = {}) {
+        let response = await this.privatePostOrderDelete (this.extend (params));
+        const data = response['data'];
+        return this.parseOrder (data);
+    }
+
+    async cancelMulitOrders (params = {}) {
```

Comment:
There's a typo here, you probably wanted to name it `cancelMultiOrders` (`Mulit` → `Multi`).
Also, we don't have a unified method with that name, but we have:

```
- cancelOrders (ids, symbol = undefined, params = {})
- cancelAllOrders (symbol = undefined, params = {})
```

↑ Therefore we need to stick with the unified API here.